integration/test_hnsw_allow_replace_deleted.py

```
@@ -290,7 +296,60 @@ def test_load_rdb_with_duplicate_label(self):
+        alive and tombstoned slots are traversed during searches and need valid
+        vectors, but previously every one with a duplicate label except the last
+        would be left with a dangling pointer. In our case, that would be slot 0
+        with the live vector. To be absolutely certain that we would be hitting the
+        dangling pointer, we will overwrite the tombstoned slot which erases the
+        label from tracked_vectors_, so that live slot is guaranteed to have a
+        dangling pointer.
+
+        In the end, the fix is all the same. Ensure that every slot corresponds with
+        a unique label on restore.
+        '''
+        server, client, logfile = self._start_server(
```

Ruff (0.16.1)

[warning] Unpacked variable `server` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)